### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by travisbrkr1234**
Bring master up-to-date

### CODE REVIEW

File: tests/testPrimeGenerator.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+var chai = require('chai');
+var expect = require('chai').expect;
+var numberToGenerate = require('../primeNumber/prime.js');
+
+describe('Generate Primes', function() {
+  beforeEach(function() {
+    console.log('Running Test: ');
```
Comment (bigdave) [inline tests/testPrimeGenerator.js]: You should be able to kill this. Mocha has different test reporters available if you want more verbose output.

File: tests/testPrimeGenerator.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  it('returns first 10 prime numbers', function() {
+    var outputArray = numberToGenerate.generatePrimes(10);
+
+    expect(outputArray).to.eql([2, 3, 5, 7, 11, 13, 17, 19, 23, 29]);
+  });
+
+  it('returns an array', function() {
+    var outputArray = numberToGenerate.generatePrimes(10);
+    expect(outputArray).to.be.a('array');
+  })
```
Comment (bigdave) [inline tests/testPrimeGenerator.js]: ;?

File: tests/testPrimeGenerator.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+})
```
Comment (bigdave) [inline tests/testPrimeGenerator.js]: ;?

File: multiples/multiplesOf3and5.js [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+  for (var number = 0; number < range; number++) {
+    if (number % 3 === 0 || number % 5 === 0) {
+      total +=number;
+    }
+  }
+
+  return total;
+}
+
+console.log('total of sums: ' + findSumOfMultiples(1000));
```
Comment (bigdave) [inline multiples/multiplesOf3and5.js]: Can you remove this and build out a test to replace it?

File: primeNumber/prime.js [hunk 5]
```diff
@@ -1,24 +1,19 @@
...
-					primes.push(currentNumber); //add the number to the primes array
-					break;
-				}
-			}
-			currentNumber++; //increment number to escape while loop
-		}
-		return primes; //return the array of prime numbers
-	}
+  if ((numberToGenerate <= 0) || (Number.isSafeInteger(numberToGenerate) === false) ) {
+    throw "please enter a positive number 1 to 100";
```
Comment (bigdave) [inline primeNumber/prime.js]: The '100' part isn't really valid anymore. Change the double quotes to singles just for consistency though. Doubles in HTML, singles in JS.

File: tests/testPrimeGenerator.js [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+  beforeEach(function() {
+    console.log('Running Test: ');
+  })
+  it('returns first 10 prime numbers', function() {
+    var outputArray = numberToGenerate.generatePrimes(10);
+
+    expect(outputArray).to.eql([2, 3, 5, 7, 11, 13, 17, 19, 23, 29]);
+  });
+
+  it('returns an array', function() {
```
Comment (bigdave) [inline tests/testPrimeGenerator.js]: This is being tested in the previous test already - you're expecting it to equal an array.

**Comment by travisbrkr1234** [comment]
Fixed these in the new commit

**Pull request closed without merging**